components/Comment.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+import moment from 'moment'
+import Container from './Container'
+
+const CommentStyled = styled.div.attrs({
+  className: 'px-2.5 py-3.75 border-b border-lightgray',
+})`
+  &:first-child {
+    border-top-width: 1px;
+  }
+`
+
+const CommentDate = styled.div.attrs({
+  className: 'font-sm text-gray font-bold mb-2.5',
+})``
+
+const CommentBody = styled.div.attrs({
+  className: 'font-sm text-black',
+})``
+
+const Comment = ({ comment, publishedAt, publishInApp }) => {
+  if (!publishInApp) return undefined
```

Comment:
is returning `undefined` the best idea?

Reply:
Have removed as I'm filtering out unpublished comments higher up! Turns out you can return null though, which is probs better than an empty div https://stackoverflow.com/questions/42083181/is-it-possible-to-return-empty-in-react-render-function